playwright/conftest.py

```diff
@@ -0,0 +1,111 @@
+"""
+Essentially holds the fixtures for each page on the website.
+
+Note that the use of context.new_page() is required when you combine multiple page objects in a single test.
+For example when generating the all_pages() fixture, we need to create a new page object for each page.
```

Comment:
I have wondered quite a lot about this 🤔  Would tests  parameterized with lists of strings, and using the function-scoped fixture `page` work ?
```python
@pytest.mark.parameterize("url", ['/foo', '/bar'])
def test_page(url, page):
  page.goto(url)
  # etc...
```
I'll push a litter helper to generate all Hugo URLs

```

Reply:
Thanks for the suggestions both here and on slack, I'm going to look into this now on this PR!

Reply:
The suggestion in slack to use `@pytest.mark.parametrize` with  `indirect=True` worked really nicely. I've now committed those changes in commit [868c5a6](https://github.com/ScilifelabDataCentre/genome-portal/pull/59/commits/868c5a6407ec0988c13d50fce5e023a7d6b727cc) 

For the helper function to generate the urls, I decided to go for a different approach (so we can use this in a GH actions test). The non-species pages are still hard-coded (as I think these are unlikely to update past launch). To search for the species pages I have kept the same approach of looking up the names of the content folders inside the Hugo site and added a check to make sure the page is not in draft mode (so not yet visible on the site). This is covered in commit [4725094](https://github.com/ScilifelabDataCentre/genome-portal/pull/59/commits/472509473dffd913ba3060980f010288daa07572) A "good enough" solution I think.